packages/rest/src/http/metadata.ts

```diff
@@ -154,7 +155,7 @@ export function gatherMetadata(
     visited.add(model);
 
     for (const property of walkPropertiesInherited(model)) {
-      if (!isVisible(program, property, visibility)) {
+      if (!isKey(program, property) && !isVisible(program, property, visibility)) {
```

Comment:
What are the implications of this? Does it mean that I can't have a "write-only" (input-only) `@key` property?